fabfile/constants.py

```diff
@@ -0,0 +1,3 @@
+DROP_CITUS = 'DROP EXTENSION IF EXISTS citus;'
+CREATE_CITUS = 'CREATE EXTENSION citus;'
+ALTER_CITUS = 'ALTER EXTENSION citus UPDATE;'
```

Comment:
If we do not provide `TO version` at the end, this will update to the latest version available. Is this really what we want?

Reply:
and maybe have aline ending at the end as well